### PR TITLE
Contact_form_update

### PR DESCRIPTION
**Pull request by MariusMalureanu**

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -4,101 +4,190 @@
...
+            font-family: Arial, Helvetica, sans-serif;
+        }
+        /*
+        .banner {
+            height: 5px;
+            width: 100%;
+            background-position: center;
+            background-repeat: no-repeat;
+        }
+        */
```
Comment (MatthiasMargot) [inline contact.html:26]: Can you delete this?

**Review by JoaoFSerra — APPROVED** [review]

File: contact.html [hunk 2]
```diff
@@ -4,101 +4,190 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Contact Form</title>
-    <link rel="stylesheet" href="contact.css">
+    <title>Contact page</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" />
+    <style>
```
Comment (MatthiasMargot) [inline contact.html:9]: Maybe you could extract this into a separate css file `contact.css` and then import it into the html via a link like so:
```
<link rel="stylesheet" href="contact.css">
```

File: contact.html [hunk 3]
```diff
@@ -4,101 +4,190 @@
...
+        
+        .image {
+            max-width: 100%;
+        }
+        
+        @media screen and (max-width: 800px) {
+            #form-area {
+                flex-direction: column;
+            }
+        }
```
Comment (MatthiasMargot) [inline contact.html:119]: This is not necessary anymore I believe

**Review by MatthiasMargot — CHANGES_REQUESTED** [review]
Could you make sure to delete code that you don't need anymore / commented out code?

**Pull request merged**